src/vm/types/bitwise_instance_def.zig

```diff
@@ -3,16 +3,18 @@ pub const CELLS_PER_BITWISE: u32 = 5;
 /// Number of input cells by bitwise operation
 pub const INPUT_CELLS_PER_BITWISE: u32 = 2;
 
+pub const TOTAL_N_BITS_BITWISE_DEFAULT = 251;
+
 /// Represents a Bitwise Instance Definition.
 pub const BitwiseInstanceDef = struct {
     const Self = @This();
 
     /// Ratio
-    ratio: ?u32,
+    ratio: ?u32 = 256,
     /// The number of bits in a single field element that are supported by the bitwise builtin.
-    total_n_bits: u32,
+    total_n_bits: u32 = TOTAL_N_BITS_BITWISE_DEFAULT,
 
-    pub fn init() Self {
+    pub fn initDefault() Self {
```

Comment:
per discussion, do we want to have conventions for structs where we have 'initDefault' iff there's a constructor involved?

Reply:
Yes following our discussion of yesterday it seems that we can avoid this one and push default values directly in the structure def.